packages/flutter_tools/lib/src/base/process.dart

```diff
@@ -164,6 +164,16 @@ String runCheckedSync(List<String> cmd, {
   );
 }
 
+/// Run cmd and return stdout on success.
+///
+/// Throws the standard error output if cmd exits with a non-zero value.
+String runSyncAndThrowStdErrOnError(List<String> cmd) {
```

Comment:
We should probably move away from "Sync" ways of running subprocesses, by the way, since they block progress meters.